### PR TITLE
add milestone details page and remove missed milestones count from al…

### PR DESCRIPTION
**Pull request by dtaylor7**
…l-data table

### CODE REVIEW

File: models/milestone.js [hunk 1]
```diff
@@ -135,6 +137,13 @@ Milestone.init({
       if (!this.getDataValue('date')) return;
       return moment(this.getDataValue('date'), 'YYYY-MM-DD').format('DD/MM/YYYY');
     }
+  },
+  updatedAt: {
+    type: DATE,
```
Comment (gds-gharper) [inline models/milestone.js:142]: This should probably be a timestamp, not a date, how does this interact with the updatedAt field below?

File: models/milestoneFieldEntryAudit.js [hunk 2]
```diff
@@ -21,7 +22,10 @@ MilestoneFieldEntryAudit.init({
   archivedAt: {
     type: DATE,
```
Comment (gds-gharper) [inline models/milestoneFieldEntryAudit.js:23]: This should probably be a TIMESTAMP, I know it's not something we changed here though

File: models/milestone.js (hunk 1 above)
Comment (dtaylor7) [reply models/milestone.js:142]: its just for reading the value from the database

Comment (dtaylor7) [reply models/milestone.js:142]: this is a timestamp, see docs: https://sequelize.org/v5/manual/data-types.html

File: models/milestoneFieldEntryAudit.js (hunk 2 above)
Comment (dtaylor7) [reply models/milestoneFieldEntryAudit.js:23]: this is a timestamp, see docs: https://sequelize.org/v5/manual/data-types.html

**Pull request merged**